src/main/java/utils/RandomUtils.java

```diff
@@ -0,0 +1,26 @@
+package utils;
+
+import java.util.Random;
+
+public class RandomUtils {
+    public String mail;
+
+    public RandomUtils(String mail) {
+        this.mail = generateRandomEmail(mail);
+    }
+
+    String generateRandomEmail(String domain){
+        int leftLimit = 48; // numeral '0'
+        int rightLimit = 122; // letter 'z'
+        int targetStringLength = 5;
+        Random random = new Random();
+
+        String generatedString = random.ints(leftLimit, rightLimit + 1)
+                .filter(i -> (i <= 57 || i >= 65) && (i <= 90 || i >= 97))
```

Comment:
It is difficult to understand the meaning of numbers `57`, `65` etc. in this filter. 
Such numbers are called Magic Numbers (you can read a short article about it here: https://www.codebyamir.com/blog/software-anti-patterns-magic-numbers#:~:text=What's%20a%20magic%20number%3F,is%20how%20it%20conveys%20intention. )

In short terms, those numbers lack meaning or intention. It would be a good idea to extract those number to a separate fields/variables and give them meaningful names.
The same can be applied to `int leftLimit = 48` where the comment would not be needed with such code:
```
int ASCII_ZERO = 48;
int leftLimit = ASCII_ZERO;
```